### DISCUSSION TITLE
No option to configure SSO for PAT; No option to enable SSO for repos

### DISCUSSION THREAD

**Discussion by nstgc** [community/community#31371] [Other Feature Feedback, Questions, & Ideas]
I've done this song and dance at BitBucket, and they have instructions that work. For reference, here are all the things I looked at:

https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on

https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization

https://ginnyfahs.medium.com/github-error-authentication-failed-from-command-line-3a545bfd0ca8

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

https://github.com/orgs/community/discussions/22786

https://github.com/orgs/community/discussions/23864

I do not have an "SSO config" button on the PAT page, nor do I have an "your organization" under my user profile menu. How am I supposed to enable PAT login from the command line, because right now I'm getting "fatal: Authentication failed for 'https://github.com/nstgc/..." errors.

**Comment by queenofcorgis** [comment]
Hi there - what type of plan do you have? SSO is only available to Enterprise accounts.
